css/homestyle.css

```diff
@@ -0,0 +1,123 @@
+body, html {
+    padding: 0;
+    margin: 0;
+    height: 100%;
+}
+.coverImage {
+    background-color: #000;
+    background: url("../Photos/hero-image 2.png") no-repeat center center fixed; 
+    background-size: cover;
+    position: relative;
+    width: 100%;
+    height: 100%;
+}
+.coverContainer {
+    width: 100%;
+    height: 100%;
+}
+.sidebar {
+    background-color:#FFFFFF;
+    top:0;
+    bottom:0;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.text {
+    font-family: Sofia Pro;
+    font-size:14px;
+    line-height:22px;
+    letter-spacing:2px;
+    padding-right: 10px;
+    padding-left:10px;
+    writing-mode: vertical-rl;
+    transform: rotate(180deg);
+}
+.dot {
+    padding-top: 25px;
+    padding-bottom: 25px;
+}
+.subtitle {   
+    font-family: CircularStd;
```

Comment:
aici la fel ca mai sus, in general se foloseste un fel de "stack" (care e o variabila defapt) care contine mai multe font-uri, daca browser-ul nu iti incarca CircularStd, poate ai alte preferinte, care e urmatorul pe care ai vrea sa il incarci. Aici nu e neaparat necesar sa pui si alte font-uri, dar ca sa stii ca idee generala.